.gitmodules

```diff
@@ -1,4 +1,6 @@
 [submodule "subiquity"]
 	path = packages/subiquity_client/subiquity
-	url = https://github.com/canonical/subiquity.git
-	branch = main
+	; url = https://github.com/canonical/subiquity.git
+	; branch = main
+	url = https://github.com/ogayot/subiquity.git
+	branch = erase-and-install
```

Comment:
Required for initial testing against the in-progress work on the Subiquity side. This needs to be removed before merging.